redis-om-spring/src/main/java/com/redis/om/spring/RedisEnhancedKeyValueAdapter.java

```diff
@@ -359,6 +411,45 @@ public void update(PartialUpdate<?> update) {
     });
   }
 
+  /*
+   * (non-Javadoc)
+   * 
+   * @see org.springframework.data.keyvalue.core.KeyValueAdapter#count(java.lang.
+   * String)
+   */
+  @Override
+  public long count(String keyspace) {
```

Comment:
`FT.SEARCH index * LIMIT 0 0` works better for this sort of thing I think as it just pulls directly from the index metadata.